tdrs-frontend/src/actions/requestAccess.js

```diff
@@ -7,28 +7,30 @@ export const SET_REQUEST_ACCESS = 'SET_REQUEST_ACCESS'
 export const SET_REQUEST_ACCESS_ERROR = 'SET_REQUEST_ACCESS_ERROR'
 export const CLEAR_REQUEST_ACCESS = 'CLEAR_REQUEST_ACCESS'
 
-export const requestAccess = ({ firstName, lastName, stt: { id } }) => async (
-  dispatch
-) => {
-  dispatch({ type: PATCH_REQUEST_ACCESS })
-  try {
-    const URL = `${process.env.REACT_APP_BACKEND_URL}/users/set_profile/`
-    const user = { first_name: firstName, last_name: lastName, stt: { id } }
-    const { data } = await (await axiosInstance).patch(URL, user, {
-      withCredentials: true,
-    })
-
-    if (data) {
-      dispatch({ type: SET_REQUEST_ACCESS })
-      dispatch({
-        type: SET_AUTH,
-        payload: { user: data },
+export const requestAccess =
```

Comment:
The prettier update caused some linter errors on this file so I used `eslint src/ --fix` to get this file to pass the linter.